### ISSUE TITLE
v1.1 tests can't restart v1.2 kube-proxy

### ISSUE THREAD

**Issue by ihmccreery** [kubernetes/kubernetes#17965]
*Services should work after restarting kube-proxy* is consistently failing in `kubernetes-upgrade-1.0-current-release-gke` (upgrades from latest 1.0 build to `master`) with:

```console
error restarting kube-proxy: couldn't restart kube-proxy: <nil> (code 1)
```

**Comment by ihmccreery** [comment]
@mikedanese:

> looks more like #17583

**Comment by mikedanese** [comment]
Just at guess that this is (old tests, new nodes, new master). I think a cherrypick of #17588 into whatever branch you are running tests in may help.

**Comment by ihmccreery** [comment]
This seems to have resolved itself.

**Comment by ihmccreery** [comment]
This is actually failing *once we upgrade the nodes to v1.2*.  Investigating...

**Comment by ihmccreery** [comment]
#20595 should fix this.

**Comment by ihmccreery** [comment]
Fixed #17965 via #20595.
